.goreleaser.yml

```diff
@@ -0,0 +1,45 @@
+before:
+  hooks:
+    - go mod download
+builds:
+-
+  id: capdctl
+  env:
+    - CGO_ENABLED=0
+  main: ./cmd/capdctl/main.go
+  binary: capdctl
+  ldflags:
+    - "-s -w -X sigs.k8s.io/cluster-api-provider-docker/cmd/versioninfo.Version={{.Version}}"
```

Comment:
also suggest updating `scripts/build-binaries.sh`

Reply:
goreleaser is replacing that so I will remove that file and add documentation to the README

Reply:
but maybe a Makefile would be good for making the binaries with these flags...I'll make an issue for that

Reply:
hmm actually yeah, let's keep build-binaries.sh around, sorry for changing my mind so quickly, but i don't want to add goreleaser as a required dependency to properly build a binary.